### ISSUE TITLE
Improve basic documentation in description

### ISSUE THREAD

**Issue by merwok** [samstav/tox-pyenv#8]
Hello!  I found this plugin when looking for doc about Python / tox usage in CircleCI.  The current doc doesn’t explain clearly *what problem* it solves, and my tox config that doesn’t use it seems to work.

**Comment by samstav** [comment]
Hi @merwok, check these out:

https://github.com/yyuu/pyenv/issues/92#issuecomment-145351621

https://github.com/samstav/circleci-python-sandbox/issues/1

**Comment by samstav** [comment]
The documentation here definitely needs updating though, thanks for pointing this out.

**Comment by merwok** [comment]
The second link is helpful, thanks!  I am using the pyenv local trick in my config, but I haven’t confirmed yet that the right versions are picked up.

**Comment by samstav** [comment]
Yeah, you might want to assert that the exact version you expect to be running is in fact running your tests. When I added these assertions and they started failing, that's when I got worried and wrote the plugin. In fact, the very tests for this plugin itself do exactly this as part of the test suite. See the environment variables set as `TOX_*`:

https://github.com/samstav/tox-pyenv/blob/master/circle.yml

Then passed through via tox config:

https://github.com/samstav/tox-pyenv/blob/master/tox.ini#L8

and the python version is asserted to be exactly as it is set during the test:

https://github.com/samstav/tox-pyenv/blob/master/test_tox_pyenv.py#L30-L52

And the check:
